docs/addons/gmaestro.md

```diff
@@ -0,0 +1,67 @@
+# gMaestro add-on for Amazon EKS Blueprints
+
+The gMaestro Blueprints AddOn deploys the gMaestro Agent on Amazon EKS using the [eks-blueprints](https://github.com/aws-quickstart/cdk-eks-blueprints) [CDK](https://aws.amazon.com/cdk/) construct.
+
+
+## Prerequisites
+Before using gMaestro, you need to:
+1. [Sign up](https://app.granulate.io/gMaestroSignup) to the gMaestro platform
+2. Download a sample YAML file - After signing up to gMaestro, navigate to the [Deploy](https://app.granulate.io/deploy) on the left-hand menu, fill in the required fields and click on "Generate Config File" 
```

Comment:
Do we have a place to download the sample YAML. If not i would add the sample YAML to this doc page.

Reply:
The only way to download it is using the instruction, I would rather not add it to the doc since it's being maintained by us and might change frequently

Reply:
I would atleast hyperlink from where to download the sample, when you say `Download a sample YAML file`

Reply:
@elamaran11 I went through the signup process, I think the doc here is clear: there is an image right below line 9 here that shows where to tap to download the file

Reply:
Oh it wasn't clear initially, it makes sense. So you mean download as shown in below image. I would rather say `Download a sample YAML file from step 3 - Download Config File` as shown below. But up to you. Im good.

Reply:
I think im good on this comment. All i was recommending at last is rephrasing your sentence to say `Download a sample YAML file from step 3 - Download Config File as shown below` and hyperlink is not needed.